src/coreclr/jit/emitarm64.cpp

```diff
@@ -16606,6 +16689,12 @@ void emitter::emitDispInsHelp(
             emitDispReg(encodingZRtoSP(id->idReg3()), size, false);    // mmmmm
             break;
 
+        case IF_SVE_GA_2A: // ............iiii ......nnnn.ddddd -- SME2 multi-vec shift narrow
+            emitDispSveReg(id->idReg1(), id->idInsOpt(), true);             // ddddd
+            emitDispSveRegList(id->idReg2(), 2, INS_OPTS_SCALABLE_S, true); // nnnn
+            emitDispImm(emitGetInsSC(id), false);                           // iiii
```

Comment:
this shows a diff when using the capstone tool:

coreclr:
```
uqrshrn z0.h, { z0.s, z1.s }, #5
sqrshrn z0.h, { z2.s, z3.s }, #16
```

capstone:
```
uqrshrn z0.h, { z0.s, z1.s }, #5
sqrshrn z0.h, { z2.s, z3.s }, #0x10
```

ie: for values >9, capstone uses 0x.

Is that something that can be fixed easily in the capstone tool?

Reply:
> Is that something that can be fixed easily in the capstone tool?

If not, should we have a (possibly) short-term "capstone quirk" mode that will output (some?) constants using capstone's model?

Reply:
It should be able to be fixed in capstone. I can make another update to it.

Reply:
@a74nh I updated capstone (same branch) to reflect that constant to display as decimal. I'm sure there are others to change, but we can update them as we go along.

Reply:
i think it will be good to show the diffs in the PR using "```diff" or some screenshot of the diff, so others can spot the difference easily.

Reply:
Please use markdown instead of screenshots. Markdown can be scraped/edited/processed; screenshots cannot.

Reply:
>I updated capstone (same branch) to reflect that constant to display as decimal.

Latest update works for me. Thanks!